src/WriterInterface.php

```diff
@@ -1,27 +1,23 @@
 <?php
-/**
- * Created by Miroslav Čillík <miro@keboola.com>
- * Date: 25/05/15
- * Time: 14:52
- */
+
+declare(strict_types=1);
 
 namespace Keboola\DbWriter;
 
 use Keboola\Csv\CsvFile;
 
 interface WriterInterface
 {
-    /** @return \PDO */
-    public function getConnection();
-    public function createConnection($dbParams);
-    public function write(CsvFile $csv, array $table);
-    public function drop($tableName);
-    public function create(array $table);
-    public function upsert(array $table, $targetTable);
-    public function tableExists($tableName);
-    public function generateTmpName($tableName);
-    public function showTables($dbName);
-    public function getTableInfo($tableName);
-    public static function getAllowedTypes();
-    public function validateTable($tableConfig);
+    public function getConnection(): \PDO;
```

Comment:
FYI. Myslím že jsou na tom i věci které nepoužívají PDO a díky tomu že tam nebyli striktní typy se to dalo různě ohnout. Možná to tedy pak bude potřeba ještě dál rozšířít.